doc/sphinx/language/gallina-extensions.rst

```diff
@@ -915,17 +915,18 @@ together, as well as a means of massive abstraction.
    Interactively defining the :n:`@module_expr_inl`\s in a series of
    :cmd:`Include` commands is equivalent to giving them all in a single
    non-interactive :cmd:`Module` command.
+
    The ! prefix indicates that any
    `Inline` directive in the type of the functor arguments will be ignored.
 
    .. todo: What is an Inline directive?  sb command but still unclear.  Maybe referring to the
-      "inline" in functor_app_annot?  or assumption_token inline assum_list?
+      "inline" in functor_app_annot?  or assumption_token Inline assum_list?
```

Comment:
I think the latter. Indeed, this documentation of the `!` prefix clearly mirrors the one of the `Inline` clause.

Reply:
This applies only to assumptions and not to these?

```
    Extraction Inline {+ @qualid }
    Print Extraction Inline
    Register Inline @qualid
    Reset Extraction Inline
```

Reply:
Interesting. I guess maybe `Register` as well, probably not the others. This would be better if it was an attribute. Anyway, to leave for further consideration.